src/Notion.ts

```diff
@@ -495,6 +498,39 @@ export class Notion {
     });
   }
 
+  /**
+ * Activates haptic motors
+ * 
+ * ```typescript
+ * notion.haptics({P7: ["tripleClick100"], P8: ["tripleClick100"]});
+ * ```
+ *
+ * @param effects Effects to produce. Must include both P7 and P8 keys in object. Each key can be an array of up to 7 commands. 
+ *  A list of commands can be found on ./utils/hapticCodes.ts - there are about 127 of them!
+ */
+  public haptics(effects: HapticEffects): any {
+    if (!this.api.didSelectDevice()) {
+      return Promise.reject(errors.mustSelectDevice);
+    }
+
+    if (!effects['P7'] || !effects['P8']) {
```

Comment:
This should be based on the available motors of the selected device. Since we are not passing that info from the OS, we can hardcode this for now, and then update to get these motor names dynamically.